### PR TITLE
[Technical Support] LPS-138168

### PR DESCRIPTION
**Pull request by cleydyr**
Related issue: https://issues.liferay.com/browse/LPS-138168
- LPS-138168 add language key
- LPS-138168 add warning to export modal
- LPS-138168 limit the number of columns and warn if needed
- LPS-138168 rephrase limitation for end user
- LPS-138168 SF

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordXLSWriter.java [hunk 1]
```diff
@@ -68,9 +68,20 @@ public DDMFormInstanceRecordWriterResponse write(
...
+			Collection<String> values = ddmFormFieldsLabel.values();
+
+			if (values.size() > _COLUMNS_MAX_COUNT) {
+				if (_log.isWarnEnabled()) {
+					_log.warn(
+						StringBundler.concat(
+							"Form has ", values.size(),
+							" fields. Due to XLS file format limitations, ",
+							"the first ", _COLUMNS_MAX_COUNT,
+							" will be included in the exported file."));
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordXLSWriter.java]: @cleydyr In this case, only the first `_COLUMNS_MAX_COUNT` will be included, right?

Comment (cleydyr) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordXLSWriter.java]: That's correct. If there are more than 256 columns (fields), then the first 256 will be included and the rest will be discarded.

Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordXLSWriter.java]: Got it, @cleydyr! thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**